manifests/charts/gateway/templates/service.yaml

```diff
@@ -39,6 +39,11 @@ spec:
     targetPort: 15017
 {{- else }}
 {{ .Values.service.ports | toYaml | indent 4 }}
+{{- end }}
+{{- if and (.Values.service.externalIPs) (ne .Values.service.type "LoadBalancer") }}
```

Comment:
For the `ne .Values.service.type "LoadBalancer"` part - what happens if both are set?  Does k8s reject, or is it just not standard?

Since it seems a bit confusing for a user to explicitly set `externalIPs` but you do not get it?

Reply:
If both are set the helm chart won't render the external IPs. According to https://istio.io/latest/docs/ops/diagnostic-tools/multicluster/ External IPs are required for istio, if you are running in an environment not support native kubernetes loadbalancer type services.
And if your environment doesn't support loadbalancer services, you shouldn't define your gateway service as type loadbalancer, but probably as nodeport service. But this is probably the wrong implementation for that nudge :)

That said I can't imagine a scenario using k8s LB service together with defined externalIPs, but maybe that's just me.
I have no idea if externalIPs work with or break LoadBalancer type services or the k8s api. 
At the end of the day it's probably better to remove the service type condition, if you want.